### PR TITLE
Update crowdinintegration.md solves issue #684

### PR DESCRIPTION
**Pull request by Loshma93**
Hi all,

I made some changes by removing unnecessary white spaces between images and making the headings bold... If you have any suggestions to update this page, please let me know.

### CODE REVIEW

**Comment by Supriyaranjan** [comment]
@Loshma93  can you post the rawgit link for this page here?

**Comment by Supriyaranjan** [comment]
@Loshma93 Have you made any changes in the images too?  In https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/684 I think mappuji meant to remove extra white space from the images to make it clearer.

**Comment by Loshma93** [comment]
@Supriyaranjan I didn't make any changes to images. I simply removed the white spaces between text and images

File: pages/crowdinintegration.md [hunk 1]
```diff
@@ -1,123 +1,109 @@
 [Google Doc](https://docs.google.com/document/d/1gsDmBSq5Tu1G_Ei1CfTRUuvvwpAWUPhHYr2UYESwISI/edit?usp=sharing)
 # Crowdin Integration
-##Objectives:
+## Objectives:
 There are two objectives for this assignment:
+
 * 1- Provide exactly one screenshot for the strings in BeLL app, and tag it.
```
Comment (mappuji) [inline pages/crowdinintegration.md]: why are you adding dash `-` after a number if the number is automagically formatted as numbering? You can try to delete the dash and you will have better number formatting. You also add an asterisk `*` before the number, I think better use number only formatting.

**Comment by dogi** [comment]
@mappuji thanks
+1 there

**Comment by mappuji** [comment]
I think someone has already solved this. See #747

**Pull request closed without merging**